content/en/apps/guides/data/users-bulk-load.md

```diff
@@ -52,13 +52,12 @@ There are three sections to the contact spreadsheet:
 
 These three columns are where you paste the results after running an import. See step 8 in "Importing users example" [below](#importing-users-example).
 1. `import.status:excluded`: This field can have three values. Over time, they should all be `imported` or `skipped` as you will have processed all users on the list:
-    <ul>
-      <li><code>imported</code> - This user has already been successfully imported</li>
-      <li><code>skipped</code> - This user was skipped</li>
-      <li><code>error</code> - Contains errors that were encountered during importation. There was an error importing see import.message:excluded field for more information</li>
-    </ul>
-3. `import.message:excluded`: The status of the last import. For example, `Imported successfully` or `Username 'mrjones' already taken`
-4. `import.username:excluded`: Use this column to ensure you're matching the response with the correct user in the contact.username to the right
+    * `<code>imported</code>` - This user has already been successfully imported
+    * `<code>skipped</code>` - This user was skipped
+    * `<code>error</code>` - There was an error importing see import.message:excluded field for more information
+    * `<code>error</code>` - Contains errors that were encountered during importation. There was an error importing see import.message:excluded field for more information
```

Comment:
You should probably drop the "<code>" tags as well.

```suggestion
    * `imported` - This user has already been successfully imported
    * `skipped` - This user was skipped
    * `error` - There was an error importing see import.message:excluded field for more information
    * `error` - Contains errors that were encountered during importation. There was an error importing see import.message:excluded field for more information
```

Also, are there supposed to be two "error" entries?

Reply:
Thanks for catching this.